util/postgres_db.py

```diff
@@ -60,6 +60,19 @@ def get_account(account_id):
             return result
 
 
+def get_account_by_email(email):
```

Comment:
If we need to have this, we should make sure it returns the same things as getAccounts above it. (teams and experiments)